services/app-web/src/s3/s3Amplify.ts

```diff
@@ -144,7 +144,7 @@ const waitFor = (delay = 1000) =>
 const retryWithBackoff = async (
     fn: () => Promise<void | S3Error>,
     retryCount = 0,
-    maxRetries = 3,
+    maxRetries = 6,
```

Comment:
If we retry 6 times how much time is that? I think this puts us over a minutes for uploading for one file if its failing. 

1. Can you please add those notes to the `scanFile` function in the code? I've been trying to document how long this can take at most per file in the code.  

 Also flagging this for design/product review- I think we should display some hint UI on our form uploads now to say "This could take up to X seconds' because we are pushing a minute.

Reply:
@haworku I see what you mean.  A known-bad file took 150 seconds to return an error.  That does seem too long.  But we found 4 retries wasn't enough, so 6 seemed safe, but it does take quite a while.

Reply:
@rswerve yeah thats fair- I do  think we need to tell users what to expect in that case

Reply:
Yeah, @mojotalantikite were kicking around ideas, like accepting the file immediately and showing the status on the summary page, or...something else.

I'm not clear on what you'd like changed or added to the PR though.

Reply:
Excuse me for adding to any confusion!  Thanks for pinging  on slack